src/features/quick-search/QuickSearchItem.js

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import { injectIntl, FormattedMessage } from 'react-intl';
 import { RecordOf } from 'immutable';
 
+import { TYPE_FILE, TYPE_FOLDER, TYPE_WEBLINK } from '../../constants';
```

Comment:
could you move this to right above the `messages` import? we usually place the `constants` import near the bottom